insights/specs/default.py

```diff
@@ -699,6 +699,7 @@ def tomcat_home_base(broker):
     tomcat_vdc_fallback = simple_command("/usr/bin/find /usr/share -maxdepth 1 -name 'tomcat*' -exec /bin/grep -R -s 'VirtualDirContext' --include '*.xml' '{}' +")
     tuned_adm = simple_command("/usr/sbin/tuned-adm list")
     udev_persistent_net_rules = simple_file("/etc/udev/rules.d/70-persistent-net.rules")
+    ulimit_hard = simple_command("/usr/bin/ulimit -a -H")
```

Comment:
Hi @jobselko, for the `simple_command` in `default.py`, an corresponding `simple_file` should be added to the `insights_archive.py`. And you can get its `simply_file` by running the command `mangle` command, e.g.:

~~~
[insights-core] # source bin/activate
[insights-core] # mangle "/usr/bin/ulimit -a -H"
insights_commands/ulimit_-a_-H
~~~

Reply:
Hi, thank you for the notice.